runtime/kusama/src/weights/xcm/mod.rs

```diff
@@ -275,3 +276,11 @@ impl<RuntimeCall> XcmWeightInfo<RuntimeCall> for KusamaXcmWeight<RuntimeCall> {
 		XcmGeneric::<Runtime>::unpaid_execution()
 	}
 }
+
+#[test]
+fn all_counted_has_a_sane_weight_upper_limit() {
+	let assets = MultiAssetFilter::Wild(AllCounted(4294967295));
+	let weight = Weight::from_ref_time(1000);
```

Comment:
Please avoid using this API. It should already be deprecated. Proof-size is just as important as ref time.